src/styles/typography.ts

```diff
@@ -14,3 +14,11 @@ export const fontByWeight = {
 export const font = (weight: keyof typeof fontByWeight): TextStyle => ({
   fontFamily: fontByWeight[weight]
 })
+
+export const typography: Record<'h1', TextStyle> = {
+  h1: {
```

Comment:
Currently the Text component also sets the color by default to themeColors.neutral, what do you think would be the best way to achieve that using this pattern? 

It seems correct to not have color as part of the typography definitions, so maybe we just specify the color manually every time?

Reply:
yeah good callouts... i think maybe it makes sense to set a default text color, ill not include it for this pr, but do some research